packages/ag-charts-community/src/chart/themes/chartTheme.ts

```diff
@@ -506,38 +517,10 @@ export class ChartTheme {
         this.params = mergeDefaults(params, this.getPublicParameters());
 
         this.config = deepFreeze(this.templateTheme(defaults));
+        this.overrides = deepFreeze(overrides);
         this.presets = deepFreeze(presets);
     }
 
-    private mergeOverrides(defaults: AgChartThemeOverrides, presets: AgPresetOverrides, overrides: AgThemeOverrides) {
```

Comment:
The merging of overrides is now handled within the graph. There are two discussion points on this:

- The removal of `zoom` and `navigator` from non-cartesian chart types is done directly in the graph with a `without()` call. It was a bit awkward trying to re-use the current `CHART_TYPE_SPECIFIC_COMMON_OPTIONS` object. Though I've noted it is something that could be improved.
- The `isPresetOverridesType()` call is no longer made. It seems like it is no longer necessary and the examples all seem to run fine without it. But I don't fully understand the problem it was solving.